tools/neo4j-admin-tools/src/main/kotlin/org/ostelco/tools/migration/Main.kt

```diff
@@ -3,13 +3,16 @@ package org.ostelco.tools.migration
 import org.neo4j.driver.v1.AccessMode
 import java.nio.file.Files
 import java.nio.file.Paths
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
 
 fun main() {
-    // neo4jExporterToCypherFile()
-    cypherFileToNeo4jImporter()
+    neo4jExporterToCypherFile("prod")
```

Comment:
comment out the call to `neo4jExporterToCypherFile`

Reply:
On the contrary, I wanted this to be uncommented just to be safe.
If both are commented then there is a risk that someone might uncomment and run the `cypherFileToNeo4jImporter()`.

This was me being explicit that use `neo4jExporterToCypherFile()`.